qiskit/quantum_info/operators/dihedral/dihedral.py

```diff
@@ -33,19 +33,19 @@ class CNOTDihedral(BaseOperator, AdjointMixin):
     """An N-qubit operator from the CNOT-Dihedral group.
 
     The CNOT-Dihedral group is generated by the quantum gates,
-    :class:`~qiskit.circuit.library.CXGate`, class:`~qiskit.circuit.library.TGate`,
-    and class:`~qiskit.circuit.library.XGate`.
+    :class:`~qiskit.circuit.library.CXGate`, :class:`~qiskit.circuit.library.TGate`,
+    and :class:`~qiskit.circuit.library.XGate`.
 
     **Representation**
 
     An *N*-qubit CNOT-Dihedral operator is stored as an affine function and a
     phase polynomial, based on the convention in references [1, 2].
 
-    The affine function consists of an NxN invertible binary matrix,
-    and an N binary vector.
+    The affine function consists of an *NxN* invertible binary matrix,
```

Comment:
A single `*` is italics is that your intent here? I might use mathmode/latex here, like:
```suggestion
    The affine function consists of an :math:`N \times N` invertible binary matrix,
```
Of course then you'd probably want to change the `*N*` elsewhere to be:
```
:math:`N`
```

Reply:
Thanks @mtreinish - I updated the docs accordingly